### PR TITLE
v1.16 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+
+### Improvements
+ - Added the ability to remove own profile picture.
+ - Changed "X" to "Cancel" on Edit Profile page.
+ - Added support for relative permalinks.
+
+### Bug Fixes
+ - Fixed an issue where the iOS app did not trigger MobileIron VPN to connect.
```
Comment (esethna) [inline CHANGELOG.md]: @enahum what about the russian translation issue? Even if the code change wasn't on this repo I think we should mention it

Comment (esethna) [reply CHANGELOG.md]: Looks good otherwise

Comment (enahum) [reply CHANGELOG.md]: Yes russian should be mentioned.

Also the VPN connection should be: `Fixed an issue where the iOS app did not waited until the on-demand VPN connection was established. (EMM Providers)` this is not a MobileIron specific case

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+ - Fixed an issue where the iOS badge notification did not always clear.
+ - Fixed an issue where the thread view displayed a new message indicator.
+ - Fixed an issue where quick multiple taps on the file icon opened multiple file previews.
+ - Fixed an issue where the settings page did not show an option to join other teams.
+ - Fixed an issue where iOS image previews didn't work after using Delete File Cache.
```
Comment (enahum) [inline CHANGELOG.md]: not iOS specific

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+ - Fixed an issue where the notification trigger word modal title was "Send email notifications" instead of "Keywords".
```
Comment (enahum) [inline CHANGELOG.md]: Android specific

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+ - Fixed an issue where the Webhook icon was misaligned and bottom edges were cut off.
+ - Fixed an issue where the user was not asked to authenticate to the app first when trying to share a photo, resulting in a white "Share modal" screen with a never-ending loading indicator.
```
Comment (enahum) [inline CHANGELOG.md]: Android specific

**Comment by amyblais** [comment]
@enahum Can you help merge this when you have a chance?

**Comment by enahum** [comment]
Waiting on @esethna approval

**Comment by amyblais** [comment]
He's off today.

**Pull request merged**